config/core/test-configs.yaml

```diff
@@ -2481,6 +2490,7 @@ test_configs:
   - device_type: rk3328-rock64
     test_plans:
       - baseline
+      - kselftest-arm64
```

Comment:
@montjoie Please confirm it's OK to enable this and baseline-nfs in BayLibre's lab

Reply:
Shouldn't that be caught by lab filters if there's any issue?

Reply:
Deciding which test plans to enable on which platform currently is a grey area between test coverage and lab capacity, so we just have to keep being pragmatic with the current system.

I think a better way forward would be to have some logic to dynamically schedule tests depending on each lab's availability, for example by limiting the number of jobs submitted to a particular device type or by querying the lab to see how long the job queue is. Then it would really become lab configuration, with test configuration only expressing what should ideally be tested.

Reply:
Sure, my thought is more that if it's a question of "can we schedule this test in this lab?" that should be being handled by the lab config. An "is it worth running this test on this board vs another board/test" question is a different thing.

Reply:
Yes, the grey area we have at the moment comes from the fact that some boards are only available in a single lab. So adding a test plan to such board can be misleading as it looks like there's some coverage for it but if the tests can't actually be run in that lab then choosing another device type would be more appropriate (if applicable).

The config schema we have now worked very well when we just had basic tests that could (should) just be enabled on all the boards. It's something we have to deal with until we come up with something more appropriate with the growing variety of tests we're adding.

Reply:
The job succeeded but there are no arm64 results: https://staging.kernelci.org/test/plan/id/623d178d0e60c006e0520902/

So it looks like there's a problem with running these tests, I haven't investigated any further.

Reply:
| No such collection 'arm64'

It looks like the image just doesn't contain the arm64 selftests.

Reply:
It looks like the kselftest build is failing, and silently discarding those tests which fail to build in a non-obvious fashion not helped by our lack of UI around kselftest build failures. There's some issues with the docker containers lagging but the main issue for several of the tests is that the kselftest makefiles don't really cope properly with O= which breaks things (the main issue is with linking multi-object executables).

Reply:
Right, so we probably should drop this PR until the arm64 kselftest build is fixed.  The kselftest make command will only return an error if all the kselftest targets failed, so it's kind of expected that these error go under the radar. We would need to parse the logs or build the targes one by one... Ideally they should all get fixed and then the logic would be to return an error if any of them fails, I think we're getting closer to this with all the fixes currently being added.